freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -65,3 +65,20 @@ restart_httpd:
     - watch:
       - file: /etc/httpd/conf.d/ipa-rewrite.conf
 
+install-healthagent:
+  cmd.run:
+    - name: /opt/salt/scripts/freeipa_healthagent_setup.sh
+    - env:
+        - FPW: {{salt['pillar.get']('freeipa:password')}}
+    - failhard: True
+    - unless: /opt/salt/scripts/freeipa_check_replication.sh
```

Comment:
do we really need a separate file for this unless?

Reply:
Yes.  Because of the '|', salt seems to have problems with it.  I tried several different permutations of escaping the character and using cmd.shell, but I kept receiving this `error:Failed: Orchestrator component went failed in 4 min(s), message: Salt execution went wrong: Rendering SLS 'base:freeipa.common-install' failed: mapping values are not allowed here; line 60 --- [...] cmd.run: - name: /opt/salt/scripts/freeipa_healthagent_setup.sh - env: - FPW: gnM3.B3J3G3l+-55j?Fe6V7CzDbdmkE_ - failhard: True - unless: ldapsearch -x -h localhost -b cn=config '(objectclass=nsds5replicationagreement)' | grep "^objectClass: nsds5replicationagreement"`